src/rust/engine/options/src/parse.rs

```diff
@@ -36,6 +40,8 @@ peg::parser! {
             = quoted_character("'")
             / escaped_character()
 
+        // NB: ##method(X) is an undocumented peg feature expression that calls input.method(pos, X)
```

Comment:
Unrelated to my change, but I was stymied by this for a few minutes, so wanted to document my findings for future readers.